src/compile/nodes/InlineComponent.ts

```diff
@@ -23,8 +23,9 @@ export default class InlineComponent extends Node {
 		super(component, parent, scope, info);
 
 		if (info.name !== 'svelte:component' && info.name !== 'svelte:self') {
-			component.warn_if_undefined(info, scope);
-			component.add_reference(info.name);
+			const name = info.name.split('.')[0]; // accommodate namespaces
```

Comment:
Did it not seem safe to just roll this logic into `warn_if_undefined` itself instead?

Reply:
Eh, I guess you need to `add_reference` here with it anyway. It might be nice though if `warn_if_undefined` didn't require an extra `name` arg as well as the `info` arg.

Reply:
It's silly to let this hold up the PR actually, gonna go ahead an approve